generate-devsite-base-classes.sh

```diff
@@ -23,14 +23,7 @@ mkdir -p ./etc
 
 cp node_modules/@google-cloud/cloud-rad/api-extractor.json .
 npx @microsoft/api-extractor run --local
-
-# install api-documenter deps manually because we're using a forked subdirectory
-cd node_modules/@microsoft/api-documenter/
-npm install --legacy-peer-deps
-npm run build
-cd ../../..
-
-npx @microsoft/api-documenter yaml --input-folder=temp
+npx @googleapis/api-documenter yaml --input-folder=temp
```

Comment:
It might make making updates to api-documenter easier if you pin this at a version range:

```
npx @googleapis/api-documenter@^7
```

Reply:
Done, thanks!